flags.go

```diff
@@ -0,0 +1,57 @@
+package fangs
+
+import (
+	"reflect"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type AddsFlags interface {
+	AddFlags(flags *pflag.FlagSet)
+}
+
+func AddCommandFlags(cmd *cobra.Command, structs ...any) {
+	AddFlags(cmd.Flags(), structs...)
+}
+
+func AddFlags(flags *pflag.FlagSet, structs ...any) {
+	for _, o := range structs {
+		addFlags(flags, false, o)
+	}
+}
+
+func addFlags(flags *pflag.FlagSet, skip bool, o any) {
```

Comment:
skip appears to be unused and inaccessible

Reply:
skip is specifically used for skipping duplicate calls for embedded structs